{"uuid": "f4fa7a5f-dd64-482d-b1af-a1b49aa49f0e", "children": ["e054b0a7-50d0-4548-b452-cac08853b2ac"], "befores": [{"name": "add_allure_env", "status": "broken", "statusDetails": {"message": "AttributeError: type object 'Dynamic' has no attribute 'environment'\n", "trace": "  File \"C:\\Users\\User\\miniconda3\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\User\\miniconda3\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\User\\miniconda3\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\miniconda3\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\miniconda3\\Lib\\site-packages\\_pytest\\fixtures.py\", line 898, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\pythonProject12\\pizzeria_tests\\conftest.py\", line 12, in add_allure_env\n    allure.dynamic.environment(host=\"pizzeria.skillbox.cc\")\n    ^^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1744229453457, "stop": 1744229453457}], "start": 1744229453457, "stop": 1744229453457}